app/src/main/java/co/nimblehq/compose/crypto/ui/screens/compose/composables/home/HomeScreen.kt

```diff
@@ -0,0 +1,70 @@
+package co.nimblehq.compose.crypto.ui.screens.compose.composables.home
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.Dimension.Dp16
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.Dimension.Dp40
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.Style
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.titleHome
+
+@ExperimentalMaterialApi
+@Composable
+fun HomeScreen() {
+    Surface {
```

Comment:
should we start a screen with [Scaffold](https://developer.android.com/jetpack/compose/layouts/material#scaffold)? 😄

Reply:
@manh-t Following the design, we don't show the AppBar or BottomNavigationBar so Scaffold is useless in this case 🤔 From what I understand, using Scaffold when we intend to use Material Components such as AppBar, BottomNavigationBar, Drawer or FAB, please correct me if I am missing something or you would suggest using Scaffold as practices? 😄

Reply:
@hoangnguyen92dn not only that, Scaffold looks like a template for a screen, even we don't use AppBar, BottomBar, etc it is still recommended to have Scaffold so later if we want to display an error message, or get the state (scaffoldState aka `context`) to do extra work 😄

Reply:
@manh-t You mean to show the error message on SnackBar, right?
> recommended to have Scaffold so later if we want to display an error message

To get the context, Jetpack Compose provided `LocalContext.current` to get the context for getting resources or stuff to to requires context. 😄

Reply:
yup, and from google examples, they also use Scaffold too. This doesn't mean we must use Scaffold, but when opening a new screen, Scaffold is more preferred than Surface, isn't it? 😁 
I just have a question. Is it okay if we start a new screen with any widget? Ie Surface, Stack, Scaffold, etc. From my insight, we should make clear on this point, so we won't face any comments like my comment in a real project. (contribute it to compass 💡 )

Reply:
@manh-t 

> yup, and from google examples, they use Scaffold too

I see because they are using Material for their UI 😄 

> I just have a question. Is it okay if we start a new screen with any widget? Ie Surface, Stack, Scaffold, etc. 

Yup. My main concern is if we use `Scaffold` but we don't apply Material components so it's necessary or not 😄 I think we could consider and it's optional in this case. 

> From my insight, we should make clear on this point, so we won't face any comments like my comment in a real project. (contribute it to compass 💡 )

Yeah... Make it clear then we would create convention for the team ✌️

Reply:
Good question, @manh-t 👍 As @hoangnguyen92dn said 👇 

> Following the design, we don't show the AppBar or BottomNavigationBar so Scaffold is useless in this case.


But We might need it when working on this 👇  implement later🙏  Let's see if we need `Scaffold`.
![Screen Shot 2565-07-11 at 11 23 54](https://user-images.githubusercontent.com/28002315/178188811-8bdb39a2-5e18-42ff-982c-b08071ab2453.png)

Reply:
Thanks for bringing this topic up @manh-t. I agree with @Wadeewee. Let's see if we need it in the following implementations 🙏  This first PR is big enough and we should give it a go? 😄